int/hwsc-user-svc/psql/0_initial_schemas.up.sql

```diff
@@ -0,0 +1,96 @@
+CREATE TYPE permission_level AS ENUM
+(
+  'NO_PERM',
+  'USER',
+  'ADMIN',
+  'USER_REGISTRATION'
```

Comment:
I prefer USER_REGISTRATION as the second element or index 1, because I will be comparing the int value. the higher the int value the higher the access level. this makes it easy to do the following:
```
if token.permission < requiredPermision {
            return "UNAUTHENTICATED
}
```